echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/pagerduty/PagerDutyNotificationService.groovy

```diff
@@ -50,17 +58,91 @@ class PagerDutyNotificationService implements NotificationService {
 
   @Override
   void handle(Notification notification) {
+    Map details = new LinkedHashMap()
```

Comment:
Mr. Groovy has left you a voice mail ... it says:

`def details = [details: notification.additionalContext.details]`

If you care about LinkedHashMap you should be able to `as LinkedHashMap`